.gitignore

```diff
@@ -0,0 +1,6 @@
+.idea/modules.xml
+.idea/php.xml
+.idea/vcs.xml
+.idea/workspace.xml
+.idea/zip.magento2.iml
+.idea/codeStyles/codeStyleConfig.xml
```

Comment:
Are there any files in the `.idea` folder that we need to manage in `git`? Or can we just ignore the whole folder by having one line like this `.idea`?